SECURITY_CONTACTS

```diff
@@ -10,5 +10,10 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
-bob
-alice
```

Comment:
Why this change?

Reply:
The old file contained dummy names (bob and alice). I figured I'd update this file while we are cleaning up OWNERS, but happy to move it to another PR too.

Reply:
I think the dummy names were intentional, as this *shouldn't* be steering in the template

Reply:
Derp, fixed! Too many repos and too many changes lol